### PR TITLE
AST-1352 - E2E test case for social icon background spacing

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for social icon background spacing

### Screenshots

https://share.getcloudapp.com/nOu5bPNK

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/social-icons/social-icon-background-spacing.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-spacing.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Social Icons in the customizer', () => {
+	it( 'social icon background spacing should apply correctly', async () => {
+		const socialIconbackSpacing = {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-spacing.test.js:5]: Variable needed in camelCase like **socialIconBackSpacing**

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-spacing.test.js]: Done

**Review by GauravKhupse — APPROVED** [review]

File: tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-spacing.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+				},
+			},
+		};
+		await setCustomize( socialIconBackSpacing );
+
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+
+		await page.waitForSelector( '.ast-header-social-1-wrap .header-social-inner-wrap .ast-builder-social-element' );
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-spacing.test.js:22]: Please verify selectors:- https://share.bsf.io/4guKrXk7
```suggestion
		await page.waitForSelector( '.ast-header-social-1-wrap .ast-builder-social-element' );
```

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-spacing.test.js]: Done

**Review by bsfsnehal — APPROVED** [review]

**Pull request merged**